recipes/flock/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: flock
+  version: "1.0"
+source:
+  git_url: https://github.com/cristhomas/immport-test.git
```

Comment:
(the sourceforge doesn't host a tarball, and using a tarball from a git repo was giving me unpacking errors.)